_layouts/rates-and-thresholds-for-employers-2017-to-2018.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="lte-ie8" lang="en"><![endif]-->
+<!--[if gt IE 8]><!-->
+<html lang="en">
+<!--<![endif]-->
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <title>Rates and thresholds for employers 2017 to 2018 - GOV.UK</title>
+
+  <!--[if gt IE 8]><!-->
```

Comment:
You can use `{% include header-assets.html %}` to not need all the stylesheet / favicon stuff